doc/HOWTO_RELEASE.rst.txt

```diff
@@ -113,7 +119,7 @@ Wine
 For building Windows binaries on OS X Wine can be used. In Wine the following
 needs to be installed:
 
-* Python 2.6-2.7 and 3.2
+* Python 2.6-2.7 and 3.3
```

Comment:
Hmm. Numpy-vendor has 2,6, 2.7, 3.2, 3.3, and 3.4. Probably we only need 2.7, 3.3, and 3.4 these days.